libraries/src/test/java/com/baeldung/unirest/HttpClientLiveTest.java

```diff
@@ -27,7 +27,7 @@
 import com.mashape.unirest.http.async.Callback;
 import com.mashape.unirest.http.exceptions.UnirestException;
 
-public class HttpClientTest {
+public class HttpClientLiveTest {
```

Comment:
@pivovarit this test was making actual calls to mocky.io, therefore, this should not be a unit test and I have named it LiveTest for the very same reason.